### PR TITLE
worked on archive functionality

### PR DESCRIPTION
**Pull request by kgajriaAgarwal**
Hello, i have implemented archive functionality using Api's . kindly review it and let me know what do think about it !
files to check- 
1. src/Helpers/Context/ArchivesContext.js
2. src/Helpers/Services/actions.js
3. src/Pages/Archives/Archives.css
4. src/Pages/Archives/Archives.js

PREVIEW : https://blithe-la-nota.netlify.app/

### CODE REVIEW

**Review by manish-GH — APPROVED** [review]
Everything looks good. The code is clean and easy to read and the UI is good as well.
Consider removing the console.log's and feel free to merge

**Review by Naman-Saxena1 — APPROVED** [review]
This is small PR, functionality is working great in preview link and code quality is also good 👍

File: src/Pages/MainPage/RouteData.js [hunk 1]
```diff
@@ -26,13 +27,10 @@ function NoMatch() {
...
-           {/* <Route exact path='/user' element={<PrivateRoute/>}>                
-              <Route exact path="/user/liked-videos"  element={<LikedVideosPage/>}/> 
-              <Route exact path='/user/playlists' element={<PlayList/>}/>
-              <Route exact path='/user/watchlater' element={<WatchLater/>}/>
-              <Route exact path='/user/history' element={<History/>}/>
+            <Route exact path='/user' element={<PrivateRoute/>}>                
+              <Route exact path="/user/archives"  element={<ArchivesPage/>}/> 
             </Route>
-            <Route exact path="/video/:videoId"  element={<Video/>}/> 
+            {/*<Route exact path="/video/:videoId"  element={<Video/>}/>
```
Comment (MayankKumar10) [inline src/Pages/MainPage/RouteData.js:33]: Suggestion: remove these comments

File: src/Pages/NotesPage/NotesPage.css [hunk 2]
```diff
@@ -1,10 +1,10 @@
 
-.main-container-wrapper{
+/* .main-container-wrapper{
```
Comment (MayankKumar10) [inline src/Pages/NotesPage/NotesPage.css]: Suggestion: remove this CSS class if its not required

**Review by MayankKumar10 — APPROVED** [review]
Everything Looks Fine just look into these suggestions you can merge it.

**Pull request merged**